src/services/volume.js

```diff
@@ -119,3 +111,52 @@ export async function cancelExpansion(params) {
     },
   })
 }
+
+export async function createRecurringJob(params) {
+  return request({
+    url: '/v1/recurringjobs',
+    method: 'post',
+    data: {
+      ...params,
+    },
+  })
+}
+
+export async function recurringJobAdd(params) {
+  return request({
+    url: params.url,
+    method: 'post',
+    data: {
+      name: params.name,
+      isGroup: params.isGroup,
+    },
+  })
+}
+
+export async function removeVolumeRecurringJob(params) {
+  return request({
+    url: params.url,
+    method: 'post',
+    data: {
+      name: params.name,
+      isGroup: params.isGroup,
+    },
+  })
```

Comment:
This is the same as the `recurringJobAdd`? I thought it should be delete action.

Reply:
They call the url is actually not the same, delete call is selectedVolume.actions.recurringJobDelete, the parameters are the same. I can combine them but the names of the functions will be very strange.

I wanted to separate `delete` and `add` so I used these two functions.